torchvision/models/resnet.py

```diff
@@ -216,11 +245,12 @@ def _forward_impl(self, x):
 
         return x
 
-    def forward(self, x):
+    def forward(self, x: Tensor) -> Tensor:
         return self._forward_impl(x)
 
 
-def _resnet(arch, block, layers, pretrained, progress, **kwargs):
+def _resnet(arch: str, block: Type[Union[BasicBlock, Bottleneck]], layers: List[int], pretrained: bool, progress: bool,
+            **kwargs: Any) -> ResNet:
```

Comment:
Could you format this with `black` to make it more legible?

Reply:
Fixed!